beekeeper-package/pom.xml

```diff
@@ -5,15 +5,15 @@
   <parent>
     <artifactId>beekeeper-parent</artifactId>
     <groupId>com.expediagroup</groupId>
-    <version>2.1.1-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>beekeeper-package</artifactId>
   <packaging>pom</packaging>
 
   <modules>
     <module>beekeeper-assembly-cleanup</module>
-    <module>beekeeper-assembly-path-scheduler-apiary</module>
+    <module>beekeeper-assembly-scheduler-apiary</module>
```

Comment:
If we're going to go through with this name change, I think we should create some tickets which propagate the name change through all of the terraform and dashboards

Reply:
Heads up that the module's `artifactId` hasn't changed (`beekeeper-assembly-path-scheduler-apiary`). We want to change this too right?

Reply:
@patduin?

Reply:
yes that should match, I actually would expect maven to complain, good catch.